index.html

```diff
@@ -9222,6 +9225,256 @@ <h3 id="example-2">
+            <code>EnvelopeFollower</code> instance.
+          </p>
+          <ul>
+            <li>Let <var>detector average</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+            <li>Let <var>compression gain</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+          </ul>
+          <p>
+            Now, the actual algorithm to determine the reduction gain, in
+            linear gain value.
+          </p>
+          <ol>
+            <li>Let <var>input signal</var> be the the render quantum of audio
+            to be processed.
+            </li>
+            <li>Let <var>threshold</var>, <var>knee</var> have the value of the
+            <a>AudioParam</a> of the same name, <a href=
+            "#db-to-linear">converted to linear unit</a> sampled at the time of
+            processing of this block (as <a>k-rate</a> parameters).
+            </li>
+            <li>Let <var>ratio</var> have the value of the <code>ratio</code>
+            <a>AudioParam</a>, sampled at the time of processing of this block
+            (as a <a>k-rate</a> parameter).
+            </li>
+            <li>Let <var>attack</var> and <var>release</var> have the value of
+            the <a>AudioParam</a> of the same name, sampled at the time of
+            processing (those are <a>k-rate</a> parameters), mutiplied by the
+            sample-rate of the <a>BaseAudioContext</a> this
+            <a>DynamicsCompressorNode</a> is <a href=
+            "#associated">associated</a> with.
+            </li>
+            <li>For each samples <var>input</var> of <var>input signal</var>,
+            execute the following steps:
+              <ol>
+                <li>Let <var>envelope rate</var> be the result of <a href=
+                "envelope-rate">computing the envelope rate</a>
+                </li>
+                <li>Let <var>attenuation</var> be the value of applying the
+                <a href="compression-curve">compression curve</a> to the
+                absolute value of <var>input</var> .
+                </li>
+                <li>Set <var>change-rate</var> to the difference between <var>
+                  attenuation</var> and <var>detector average</var>. Multiply
+                  <var>change-rate</var> by the <var>envelope rate</var>. Add
+                  <var>change-rate</var> to <var>detector average</var>, and
+                  clamp <var>detector average</var> to a maximum of 1.0.
+                </li>
+                <li>If attacking, set <var>attack</var> to <var>detector
+                average</var> minus <var>compressor gain</var>. Set
+                <var>compressor gain</var> to the multiplication of
+                <var>attack</var> by <var>envelope rate</var>. Clamp
+                <var>compressor gain</var> to a maximum of 1.0.
+                </li>
+                <li>Else, multiply <var>compressor gain</var> by <var>envelope
```

Comment:
Mistake: compressor gain is not set to the product of attack and envelope rate, it is increased by that amount. Could reword as follows:

Multiply _attack_ by _envelope rate_. Add attack to _compressor gain_.